sdk/remoterendering/azure-mixedreality-remoterendering/swagger/AUTOREST.md

```diff
@@ -0,0 +1,191 @@
+# Azure Remote Rendering Service client library for Python
+
+## Setup
+
+```ps
+npm install -g autorest
+```
+
+## Generation
+
+```ps
+cd <swagger-folder>
+autorest AUTOREST.md
+```
+
+### Code generation settings
+
+```yaml
+title: RemoteRenderingRestClient
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/92aa7802beffd6bdae18b808e45c4897de2cb2e2/specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json
+output-folder: ../azure/mixedreality/remoterendering/_generated
+namespace: azure.mixedreality.remoterendering._generated
+no-namespace-folders: true
+license-header: MICROSOFT_MIT_NO_VERSION
+enable-xml: false
+clear-output-folder: true
+vanilla: true
+python: true
+v3: true
+no-async: false
+```
+
+### Directive renaming conversion "creationTime" property to "createdOn"
+
+``` yaml
+directive:
+    from: swagger-document
+    where: '$.definitions.conversion.properties.creationTime'
+    transform: >
+        $["x-ms-client-name"] = "createdOn";
+```
+
+### Directive renaming conversion to asset_conversion
+
+``` yaml
+directive:
+    from: swagger-document
+    where: '$.definitions.conversion'
+    transform: >
+        $["x-ms-client-name"] = "asset_conversion";
+```
+
+### Directive renaming create_conversion_settings to create_asset_conversion_settings
+
+``` yaml
+directive:
+    from: swagger-document
+    where: '$.definitions.create_conversion_settings'
+    transform: >
+        $["x-ms-client-name"] = "create_asset_conversion_settings";
+```
+
+### Directive renaming conversion output outputAssetUri property to assetUri
+
+``` yaml
+directive:
+    from: swagger-document
+    where: '$.definitions.conversion.properties.output.properties.outputAssetUri'
+    transform: >
+        $["x-ms-client-name"] = "assetUri";
+```
+
+### Directive renaming conversion_input_settings to asset_conversion_input_settings
+
+``` yaml
+directive:
+    from: swagger-document
+    where: '$.definitions.conversion_input_settings'
+    transform: >
+        $["x-ms-client-name"] = "asset_conversion_input_settings";
+```
+
+### Directive renaming conversion_output to asset_conversion_output
+
+``` yaml
```

Comment:
This type seems to be defined in-line in the REST API, so I think this directive doesn't work as intended.

Reply:
correct - this is a leftover from when i pulled the definition out in the swagger file. since i abandoned this endeavor this does not to anything anymore. will remove

Reply:
You could see if this works:

```yaml
directive:
  - from: swagger-document
    where: $.definitions
    transform: >
      $["AssetConversionOutput"] = $.conversion.properties.output;
      $.conversion.properties["output"] = {"$ref": "#/definitions/AssetConversionOutput"};
```

Reply:
i'll give it a try